Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -465,7 +476,14 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
     /// Hides typing Indicator
     /// - Parameter typingMembers: typing members gotten from `channelController`
     open func hideTypingIndicator() {
+        guard !typingIndicatorView.isHidden else { return }
```

Comment:
`guard typingIndicatorView.isVisible else { return }` ? 🤯

Reply:
Or maybe just `if typingIndicatorView.isHidden { return }`. The `isVisible` extension is a candidate for people forgetting to use it and making the code inconsistent, I didn't even know we had this extension 😅

Reply:
Done ✅